package.json

```diff
@@ -15,7 +15,6 @@
     "@react-three/drei": "^9.77.3",
     "@react-three/fiber": "^8.13.3",
     "axios": "^1.4.0",
-    "bcrypt": "^5.1.0",
```

Comment:
Probably should keep this for hashing

Reply:
sure, I didn't know if we wanted to keep it since PWs are hashed by firebase but we can use it to hash other info.